test/e2e/ig_latency.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"runtime"
+	"time"
+)
+
+const NumTestTimes = 4
+
+func TestWithImportGroupLatency(
+	apiName string,
+	opt *TestOption,
+	testSuite func(*TestOption, bool)) error {
+	if LatencyTest {
+		return latencyTest(apiName, opt, testSuite)
+	}
+	goldenTest(opt, testSuite)
+	return nil
+}
+
+func goldenTest(opt *TestOption, testSuite func(*TestOption, bool)) {
```

Comment:
since goldenTest is here, maybe change filename from ig_latency.go to something general indicating this holds various test funcs

Reply:
Good point. Renamed to test_driver.go